test/hex_tarball_SUITE.erl

```diff
@@ -224,6 +315,10 @@ unpack_error_handling_test(_Config) ->
       "contents.tar.gz" => <<"badtar">>,
       "CHECKSUM" => <<"63E0D44ED4F61F5A1636A516A6A26890052CE0BB1B1A6EDC66C30282E2EC1A58">>
     },
+
+    {error, {tarball, {inner_checksum_mismatch, _, _}}} = unpack_files(Files5),
+
+    %% I am not sure how to trigger this with the metadata changes. It may be an obsolete check -- Bryan
```

Comment:
See comment

Reply:
It can only happen for an invalid tarball but the checksum needs to match. Maybe it will work with the following checksum:

```elixir
iex(1)> :crypto.hash(:sha256, "badtar") |> Base.encode16()
"8CFEB690D349787B969F3485DE74D3F18FDB877B69C4D29B0E3B09C9F0E4F503"
```

Reply:
Will checkout 👍

Reply:
aye, that works.